styles.css

```diff
@@ -0,0 +1,67 @@
+* {
+    font-family: sans-serif;
+  }
+  
+  #navbar {
+    background-color: rgba(3, 25, 20, 0.834);
+    padding: 40px;
+    display: flex;
+    justify-content: flex-end;
+    gap: 30px;
+    top: 0;
+    overflow: hidden;
+    position: sticky;
+}
+.nav-links {
+    text-decoration: none;
+    color: white;
+    font-size: 20px;
+    font-weight: 700;
+  }
+  a:hover {
+    color: cyan;
+  }
+  
+#welcome-section {
+    height: 100vw;
+    background-color: rgb(86, 133, 105);
+    text-align: center;
+    padding: 17%;
+  }
+  #welcome-section h1 {
+    font-size: 50px;
+    color: white;
+  }
+  #welcome-section p {
+    font-size: 20px;
+    color: rgb(193, 186, 127);
+  }
+  #projects {
+    text-align: center;
+    padding: 30px;
+    padding-top: 0;
+  }
+  #contact {
+    height: 50px;
+    background-color: rgb(47, 46, 46);
+    color: white;
+    text-align: center;
+    padding: 17%;
+  }
+
+ .project-tile{
+  position:relative;
+  text-align: center;
+  display:flex;
+  border: 2px solid rgb(43, 43, 43);
+  background-color: gray;
+  border-radius: 5px;
+  padding: 2px;
+ }
```

Comment:
You did good work but still, your design has some little issues. Kindly fix the CSS so that the navbar is  always appears at the top of the page.
![Untitled](https://user-images.githubusercontent.com/114990830/235443525-1d3e6aa5-96fa-4318-ae5f-74b5481348f1.png)